controller/metrics/metrics_test.go

```diff
@@ -112,7 +112,7 @@ status:
     status: Healthy
 `
 
-var noOpHealthCheck = func() error {
+var noOpHealthCheck = func(r *http.Request) error {
```

Comment:
This input parameter `r *http.Request` is not used in this function. Any reason it needs to be defined?